javascript-functions/main.js

```diff
@@ -0,0 +1,35 @@
+// console.log('hi');
+
+function convertMinutesToSeconds(minutes) {
+  var answer = minutes * 60;
+  return answer;
+}
+var converted = convertMinutesToSeconds(5);
+console.log('converted answer: ', converted);
+
+function greet(name) {
+  var greeting = 'Hey ' + name;
+  return greeting;
+}
+var greeted = greet('Beavis');
+console.log('greeting: ', greeted);
+
+function getArea(width, height) {
+  var answer = width * height;
+  return answer;
+}
+var three = getArea(17, 42);
+console.log('width x height: ', three);
+
+function getFirstName(person) {
+  var answer = person.firstName;
+  return answer;
+}
+var four = getFirstName({ firstName: 'Lelouche', lastName: 'Lamperouge' });
+console.log('first name: ', four);
+
+function getLastElement(array) {
+  return array[array.length - 2];
```

Comment:
this should be array.length-1 good sir!

Reply:
![Screen Shot 2022-02-17 at 7 58 01 PM](https://user-images.githubusercontent.com/55678486/154614938-c27913cf-ed49-4ba8-a9f2-b8f530433a87.png)
whoops, thanks Brett!